Bio/Sequencing/Applications/_samtools.py

```diff
@@ -430,10 +429,9 @@ class SamtoolsMergeCommandline(AbstractCommandline):
     >>> from Bio.Sequencing.Applications import SamtoolsMergeCommandline
     >>> out_bam = "/path/to/out_bam"
     >>> in_bam = ["/path/to/input_bam1", "/path/to/input_bam2"]
-    >>> samtools_merge_cmd = SamtoolsMergeCommandline(\
-                                                      out_bam=out_bam,\
-                                                      input_bam=in_bam)
-    >>> print(samtools_merge_cmd)
+    >>> merge_cmd = SamtoolsMergeCommandline(out_bam=out_bam,
+    ...                                               input_bam=in_bam)
```

Comment:
Indentation seems off.

Reply:
It does look wrong on GitHub, I agree. Maybe emacs stuck a tab in there? I must check that...